### DISCUSSION TITLE
One-time extension of the development time for 24.12

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/sig-release#880] [Announcements]
Dear Eclipse Tractus-X Community,

We would like to propose an extension of the current **Development Phase** for the Eclipse Tractus-X 24/12 release by **two weeks**. This means that the **Feature Freeze and E2E Test Phase** would also shift accordingly, as would the **Finalization Phase** (for activities like the review of QGate issues and creation of the overall changelog).

**Important Note:**
The proposed extension **will not affect the release date of Eclipse Tractus-X 24.12**. This extension would simply provide two additional weeks for development work.

One side effect of this change would be a slight overlap of the **Finalization Phase for 24.12** with the **Development Phase for 25.03.** We have created a diagram for reference, which outlines the phases, milestones, and changes. You can view it here:
[Phase and Milestone Diagram](https://github.com/orgs/eclipse-tractusx/projects/26/views/35)

Moved phases/milestones are signed with (NEW)

**What We Need From You:**

We are seeking feedback from the community to gauge whether this extension would be acceptable and beneficial. Please feel free to:

	•	**Vote:** Let us know if you support this extension.
	•	**Comment:** Share your thoughts, concerns, or suggestions about the proposed timeline shift.

We aim to finalize a decision by **Friday this week during the office hour.** Your input is crucial to ensuring this decision aligns with the needs of the community.

Thank you for your collaboration!

**Comment by stephanbcbauer** [comment]
I am fine (please thumbs up)

**Reply by ybidois** [reply]
More time to dev our features is great! 🙏

**Comment by stephanbcbauer** [comment]
I am NOT fine (please also add comments)

**Comment by AnuragNagpure** [comment]
I am fine (please thumbs up)
